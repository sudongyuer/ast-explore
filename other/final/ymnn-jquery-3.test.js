import pluginTester from 'babel-plugin-tester'
import ymnnJquery from './ymnn-jquery-3'

pluginTester({
  plugin: ymnnJquery,
  tests: [
    {code: 'foo.hide();', snapshot: false},
    {code: `$(el).hide();`, snapshot: true},

    {code: 'bar.show();', snapshot: false},
    {code: `$(el).show();`, snapshot: true},

    {code: `foo.addClass(otherClassThing);`, snapshot: false},
    {code: `$(el).addClass(className);`, snapshot: true},
  ],
})
