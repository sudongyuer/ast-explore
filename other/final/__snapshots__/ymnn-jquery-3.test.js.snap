// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`2. ymnn-jquery 1`] = `
"
$(el).hide();

      ↓ ↓ ↓ ↓ ↓ ↓

el.style.display = \\"none\\";
"
`;

exports[`4. ymnn-jquery 1`] = `
"
$(el).show();

      ↓ ↓ ↓ ↓ ↓ ↓

el.style.display = \\"\\";
"
`;

exports[`6. ymnn-jquery 1`] = `
"
$(el).addClass(className);

      ↓ ↓ ↓ ↓ ↓ ↓

el.classList.add(className);
"
`;
